### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1798,6 +1824,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 "checksum clicolors-control 1.0.1 (registry+https://github.com/rust-lang/crates.io-index)" = "90082ee5dcdd64dc4e9e0d37fbf3ee325419e39c0092191e0393df65518f741e"
 "checksum cloudabi 0.0.3 (registry+https://github.com/rust-lang/crates.io-index)" = "ddfc5b9aa5d4507acaf872de71051dfd0e309860e88966e1051e462a077aac4f"
 "checksum console 0.8.0 (registry+https://github.com/rust-lang/crates.io-index)" = "b147390a412132d75d10dd3b7b175a69cf5fd95032f7503c7091b8831ba10242"
+"checksum console 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)" = "62828f51cfa18f8c31d3d55a43c6ce6af3f87f754cba9fbba7ff38089b9f5612"
```
Comment (matklad) [inline Cargo.lock]: That's unfortunate, but it seems it's  simpler to upgrade than to wait for the dupes to be resolved...

**Comment by matklad** [comment]
well, it's been pending for quite a while now, and, if we extrapolate that,
I won't expect the fix soon :)

**Comment by lnicola** [comment]
> well, it's been pending for quite a while now, and, if we extrapolate that,

By the way, https://github.com/rust-analyzer/ra_vfs/pull/16? :joy_cat:

**Pull request merged**